kroxylicious-filters/kroxylicious-multitenant/src/test/java/io/kroxylicious/proxy/filter/multitenant/MultiTenantFilterTest.java

```diff
@@ -152,9 +167,111 @@ void responseTransformed(@SuppressWarnings("unused") String testName, ApiMessage
     }
 
     @Test
-    void testContributor() {
-        MultiTenantTransformationFilterFactory factory = new MultiTenantTransformationFilterFactory();
-        assertThat(factory.createFilter(Mockito.mock(FilterFactoryContext.class), Mockito.mock(MultiTenantConfig.class)))
-                .isInstanceOf(MultiTenantTransformationFilter.class);
+    void shouldReWriteTopic() {
+        var multiTenantTransformationFilter = new MultiTenantFilter(new MultiTenantConfig(null));
```

Comment:
Do we want a copy if this with the old class name to validate compatibility?

Reply:
Not sure I follow.   This class is exclusively concerned with testing the filter (which is now package private), rather than the factory.

I added a io.kroxylicious.proxy.filter.multitenant.MultiTenantTest#createFilterDeprecatedFactory and io.kroxylicious.proxy.multitenant.MultiTenantIT#deprecatedFactoryName to make sure that support for the old factory name is preseved.